src/ansys/acp/core/_tree_objects/lookup_table_column_base.py

```diff
@@ -88,17 +88,17 @@ def __init__(
             self, value_type
         )
 
-        self.dimension_type = dimension_type
+        self.physical_dimension = physical_dimension
         if data is not None:
             self.data = data
 
     value_type = grpc_data_property_read_only(
         "properties.value_type", from_protobuf=lookup_table_column_value_type_from_pb
     )
-    dimension_type = grpc_data_property(
+    physical_dimension = grpc_data_property(
```

Comment:
Comment that a `physical_dimension` is better than `dimension_type` from a customer point of view and therefore we decided to use a different name in the frontend.